src/terra-dev-site/contributing/DeveloperWorkflow.c.contributing.md

```diff
@@ -98,64 +100,43 @@ npm run lint:js
-
-## Testing Components
-
-Terra UI has 2 types of tests:
-* Unit tests, using Jest
-* Browser tests, using Webdriver.io
+We recommend that you install a plugin to your editor to run [ESLint](https://eslint.org/docs/user-guide/integrations#editors) and [Stylelint](https://stylelint.io/user-guide/complementary-tools/#editor-plugins) automatically when you type, which will then show you any errors inline.
 
-### Jest
-We use Jest for testing, including unit tests for functionality and snapshot testing for components.
+## Running Jest Tests
 
-If you make intentional changes to an existing component, you will need to update its snapshot:
-npm test -- -u
+For Terra UI, we use [Jest](https://jestjs.io/) for testing, including unit tests for functionality and snapshot testing for components.
 
-At minimum, all components should have a "does it render" test:
+Run the following command in the root directory to run Jest for all packages.
 
-```jsx
-import React from 'react';
-import ReactDOM from 'react-dom';
-import Link from './Link';
+`npm run jest`
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<Link />, div);
-});
-```
+You can also run Jest in the individual package directories.
 
+Change directories to the specific package in your terminal, e.g. `cd packages/terra-button`, and run the following command
 
-You may also want to test class level functions:
+`npm run test:jest`
 
-```jsx
-it('tracks the number of clicks', () => {
-  const div = document.createElement('div');
-  const instance = ReactDOM.render(<Link />, div);
+If you make intentional changes to an existing component, you will need to update its snapshot:
 
-  t.equals(instace.state.clicks, 0);
-  instance.trackClick();
-  t.equals(instace.state.clicks, 1);
-});
-```
+`npm run test:jest -- -u`
 
-Be careful when adding tests; we need to be wary of when we're effectively testing the React internals as opposed to our own components.
+## Running Webdriver.io (WDIO) Tests
 
-#### Running Jest Tests
+For Terra UI, we use also use [Webdriver.io](http://webdriver.io/) for testing, including browser tests for functionality and visual regression testing and accessibility testing for components.
 
-// TODO
+Run the following command in the root directory to run Webdriver.io for all packages.
 
-### Webdriver.io (WDIO)
+`npm run wdio`
 
-// TODO
+You can also run Webdriver.io in the individual package directories.
 
-#### Visual Regression Testing
+Change directories to the specific package in your terminal, e.g. `cd packages/terra-button`, and run the following command
 
-// TODO
+`npm run test:wdio`
 
-#### Running Webdriver Tests
+More info on writing webdriver.io tests for terra can be [found here](https://github.com/cerner/terra-toolkit/blob/master/docs/Wdio_Utility.md).
```

Comment:
It might be valuable to add information on [wdio cli options](http://webdriver.io/guide/testrunner/gettingstarted.html). When combined with piping `-- -flag`, cmd line options open up to running specific files from root, etc.